spec/spec_helper.rb

```diff
@@ -3,9 +3,7 @@
 
 require 'rspec'
 require 'agendor-ruby'
-require 'codeclimate-test-reporter'
 
```

Comment:
this method of reporting to CC has been deprecated :(

https://github.com/codeclimate/ruby-test-reporter/blob/master/README.md